sphinx_gallery/backreferences.py

```diff
@@ -61,47 +61,56 @@ def visit_Attribute(self, node):
             self.visit(node)
 
     def get_mapping(self):
+        imported_names_split = \
+            [key.split('.') for key in self.imported_names.keys()]
+        if imported_names_split:
+            max_import_splits = len(max(imported_names_split, key=len))
+        else:
+            max_import_splits = 1
```

Comment:
to avoid the conditional you could do:
```
# ensure that the max is at least one here
max_import_splits = len(max(imported_names_split + [''], key=len))
```